app/templates/check_results.html

```diff
@@ -259,6 +259,55 @@ <h3>
                             </table>
                         </div>
                     </div>
+
+                    <div class="card p-1 bg-light border rounded-3 mb-3">
+                        {{ card_header("TLS: Mail Exchange", result.domain.tls.mail_transfer) }}
```

Comment:
there are some missing translations in the Polish version:
![image](https://github.com/CERT-Polska/mailgoose/assets/1233067/580c005d-e690-4bc3-a94b-e33d1ac407b9)